.coafile

```diff
@@ -39,7 +39,7 @@ max_lines_per_file = 1000
 
 [rst]
 bear = reSTCheckBear
-files = README.rst
+files = **/*.rst
```

Comment:
Isnt `**/*.rst` collecting root directory members a bug?

My guess is the glob is being run on absolute-path filenames, instead of project-root-relative filenames.